src/main/java/se/sundsvall/contract/api/ContractResource.java

```diff
@@ -104,29 +111,59 @@ ResponseEntity<List<Contract>> getContracts(
+			@ApiResponse(
+				responseCode = "200",
+				description = "Ok",
+				useReturnTypeSchema = true),
+			@ApiResponse(
+				responseCode = "404",
+				description = "Not Found",
+				content = @Content(schema = @Schema(implementation = Problem.class)))
 		})
-	@GetMapping(path = "/{id}", produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
+	@GetMapping(path = "/{contractId}", produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
 	ResponseEntity<Contract> getContractById(
 		@Parameter(name = "municipalityId", description = "Municipality id") @ValidMunicipalityId @PathVariable("municipalityId") final String municipalityId,
-		@Parameter(description = "Contract id") @PathVariable("id") final String id) {
-		return ok(service.getContract(municipalityId, id));
+		@Parameter(description = "Contract id") @PathVariable("contractId") final String contractId) {
+		return ok(service.getContract(municipalityId, contractId));
 	}
 
 	@Operation(
 		summary = "Update a contract",
 		responses = {
-			@ApiResponse(responseCode = "200", description = "Ok"),
-			@ApiResponse(responseCode = "404", description = "Not Found")
+			@ApiResponse(
+				responseCode = "200",
+				description = "Ok"),
+			@ApiResponse(
+				responseCode = "404",
+				description = "Not Found",
+				content = @Content(schema = @Schema(implementation = Problem.class)))
 		})
-	@PutMapping(path = "/{id}", consumes = APPLICATION_JSON_VALUE, produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
+	@PutMapping(path = "/{contractId}", consumes = APPLICATION_JSON_VALUE, produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
 	ResponseEntity<Void> updateContract(
 		@Parameter(name = "municipalityId", description = "Municipality id") @ValidMunicipalityId @PathVariable("municipalityId") final String municipalityId,
-		@Parameter(description = "Contract id") @PathVariable("id") final String id,
+		@Parameter(description = "Contract id") @PathVariable("contractId") final String contractId,
 		@RequestBody @Valid final Contract contract) {
 
-		service.updateContract(municipalityId, id, contract);
+		service.updateContract(municipalityId, contractId, contract);
+		return ok().build();
+	}
+
+	@Operation(
+		summary = "Delete a contract",
+		responses = {
+			@ApiResponse(
+				responseCode = "202",
+				description = "Accepted"),
+			@ApiResponse(
+				responseCode = "404",
+				description = "Not Found",
+				content = @Content(schema = @Schema(implementation = Problem.class)))
+		})
+	@DeleteMapping(path = "/{contractId}", produces = APPLICATION_PROBLEM_JSON_VALUE)
+	ResponseEntity<Void> deleteContract(
+		@Parameter(name = "municipalityId", description = "Municipality id") @ValidMunicipalityId @PathVariable("municipalityId") final String municipalityId,
+		@Parameter(description = "Contract id") @PathVariable("contractId") final String contractId) {
+		service.deleteContract(municipalityId, contractId);
 		return ok().build();
```

Comment:
`accepted().build()` ? At least according to your documented response-code for this resource/method

(Isn't HTTP 204 more appropriate for deletes, b.t.w.?)

Reply:
oops, you're right, forgot to change. 
DIGG said 202 or 204, but I agree. Since we can deliver a result on the operation 204 is more appropriate.